### ISSUE TITLE
Multiple triggers

### ISSUE THREAD

**Issue by ReDetection** [edospadoni/remote-band#1]
I noticed that accidental device changes results in message duplicates – if I choose my device twice, I get two triggers per actual message.

Not a problem for me now, but definitely a bug. Maybe I'll take a look at it later

**Comment by edospadoni** [comment]
Yes it's a bug, I can reproduce it. At the moment is not a problem also for me but if you want to fix it feel free to open a pull request. Thank you
